src/controllers/hid/hidenumerator.cpp

```diff
@@ -65,23 +65,32 @@ HidEnumerator::~HidEnumerator() {
 QList<Controller*> HidEnumerator::queryDevices() {
     qInfo() << "Scanning USB HID devices";
 
+    QStringList enumeratedDevices;
     hid_device_info* device_info_list = hid_enumerate(0x0, 0x0);
     for (const auto* device_info = device_info_list;
             device_info;
             device_info = device_info->next) {
         auto deviceInfo = mixxx::hid::DeviceInfo(*device_info);
+        // The hidraw backend of hidapi on Linux returns many duplicate hid_device_info's from hid_enumerate,
+        // so filter them out.
+        if (enumeratedDevices.contains(deviceInfo.pathRaw())) {
+            qInfo() << "Duplicate HID device, excluding" << deviceInfo;
```

Comment:
Do we need to verify that the contents in `deviceInfo` also match that of the already found device? Do you have more information why this happens?

Reply:
I do not know why this occurs. I'm not sure if it's a bug or a feature of the hidraw implementation. I suppose it would be a good idea to ask upstream. We do not need to verify the rest of the `deviceInfo`; the raw path is sufficient to filter duplicate devices.

Reply:
https://github.com/libusb/hidapi/issues/298

Reply:
Please add the link to this issue in the comment so that we can keep track of it.

Reply:
done b5904a7